core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1072,7 +1164,7 @@ write_files:
             - replicationcontrollers
             verbs:
             - get
-          - apiGroups: ["policy"]
+          - apiGroups: [""]
```

Comment:
Just for confirmation but you've changed this so that it refers to an existing api resource?
I thought so because I could see no api resource named `pods/eviction` in the [`policy` api group](https://github.com/kubernetes/kubernetes/blob/master/api/swagger-spec/policy.json).